content/en/docs/migrating/_index.md

```diff
@@ -0,0 +1,835 @@
+---
+title: "Migrating to the AWS SDK for Go V2"
+linkTitle: "Migrating"
+description: "How to migrate to the AWS SDK for Go V2 from AWS SDK for Go V1."
+---
+
+## Minimum Go Version
+
+The {{% alias sdk-go %}} requires a minimum version of Go 1.15. Migration from AWS SDK for Go to {{% alias sdk-go %}}
+might require you to upgrade your application by one or more Go versions. The latest version of Go can be downloaded on
+the [Downloads](https://golang.org/dl/) page. See the [Release History](https://golang.org/doc/devel/release.html) for
+more information about each Go version release, and relevant information required for upgrading.
+
+## Modularization
+
+The {{% alias sdk-go %}} has been updated to take advantage of the Go modules which became the default development mode
+in Go 1.13. A number of packages provided by the SDK have been modularized and are independently versioned and
+released respectively. This change enables improved application dependency modeling, and enables the SDK to
+provide new features and functionality that follows the Go module versioning strategy.
+
+The following list are some of the Go modules provided by the SDK:
+Module | Description
+--- | ---
+`github.com/aws/aws-sdk-go-v2` | The SDK core
+`github.com/aws/aws-sdk-go-v2/config` | Shared Configuration Loading
+`github.com/aws/aws-sdk-go-v2/credentials` | AWS Credential Providers
+`github.com/aws/aws-sdk-go-v2/ec2imds` | {{% alias service=EC2 %}} Instance Metadata Service Client
+`github.com/aws/aws-sdk-go-v2/service/` | Service Client Modules
+`github.com/aws/aws-sdk-go-v2/feature/` | High-Level utilities for services, for example the {{% alias service=S3 %}} Transfer Manager
+
+## Configuration Loading
+
+The [session]({{< apiref v1="aws/session" >}}) package and associated functionality are replaced with a simplified
+configuration system provided by the [config]({{< apiref "config" >}}) package. The `config` package is a separate Go
+module, and can be included in your applications dependencies by performing
+`go get github.com/aws/aws-sdk-go-v2/config`.
+
+[session.New]({{< apiref v1="aws/session#New" >}}), [session.NewSession]({{< apiref v1="aws/session#NewSession" >}}),
+[NewSessionWithOptions]({{< apiref v1="aws/session#NewSessionWithOptions" >}}), and
+[session.Must]({{< apiref v1="aws/session#Must" >}}) must be migrated to
+[config.LoadDefaultConfig]({{< apiref "config#LoadDefaultConfig" >}}).
+
+The `config` package provides several helper functions that aid in overriding the shared configuration loading
+programmatically. These function names are prefixed with `With` followed by option that they override. Let's look at
+some examples of how to migrate usage of the `session` package. For more information on loading shared configuration
```

Comment:
The sentence starting "For more information" should be in a separate paragraph (a Note perhaps).